misk/src/main/kotlin/misk/web/proxy/WebProxyAction.kt

```diff
@@ -49,19 +50,16 @@ class WebProxyAction : WebAction {
   @Inject private lateinit var entries: List<WebProxyEntry>
   @Inject @JvmSuppressWildcards private lateinit var clientRequest: ActionScoped<Request>
 
-  private val plainTextMediaType = MediaTypes.TEXT_PLAIN_UTF8.asMediaType()
-
   @Get("/{path:.*}")
   @Post("/{path:.*}")
   @RequestContentType(MediaTypes.ALL)
   @ResponseContentType(MediaTypes.ALL)
-  @Unauthenticated
+  @Unauthenticated  // TODO(adrw) should this be unauthenticated?
```

Comment:
nope. It doesn’t run on the prod servers so it doesn’t need it.

Reply:
#429